src/parse.js

```diff
@@ -533,6 +533,34 @@ function flattenKindShorthand(result, tag, key) {
   }
 }
 
+/**
+ * Retreive description next to specific tags.
+ * @param {string} rawComment raw comment string
+ * @param {string[]} tags tags to consider
+ * @returns description string
+ */
+function retreiveDescriptionNextToTags(rawComment, tags) {
+  // regex to retreive the whole description string (new lines included)
+  const descLines = new RegExp(
```

Comment:
can it be just `starts with *` and ends either with `*/` or `* @` (new tag start)?

Reply:
you are matching all the description lines from JSDoc, right?

```
/**
 * @tag xxx
 * <MATCHED_CONTENT>
 * @other_tag
 */
```